x-pack/plugins/actions/server/routes/find.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema, TypeOf } from '@kbn/config-schema';
+import {
+  IRouter,
+  RequestHandlerContext,
+  KibanaRequest,
+  IKibanaResponse,
+  KibanaResponseFactory,
+} from 'kibana/server';
+import { FindOptions } from '../../../../legacy/plugins/alerting/server/alerts_client';
+import { LicenseState } from '../lib/license_state';
+import { verifyApiAccess } from '../lib/license_api_access';
+
+// config definition
+const querySchema = schema.object({
+  per_page: schema.number({ defaultValue: 20, min: 0 }),
```

Comment:
is `0` right here as `min`?  To cope with some kind of edge case from the caller, or it means something special in ES?